from-scratch/README.md

```diff
@@ -101,6 +101,34 @@ Read `high-score-tables` for a completed example of `vite.config.js`.
 
 You know how sometimes you get code review telling you to remove files like /node_modules or .vscode settings? Vite has created a `.gitignore` file for you. Have a look at it and see if you agree with what it's ignoring. You can add more files to ignore if you want.
 
+## Eslint and Prop-types
+
+New Vite projects may encounter issues with Eslint, displaying red marks below some variables. This occurs because the default Eslint configuration requires prop-types in React projects. Here's how to resolve it:
```

Comment:
```suggestion
Eslint might display red marks below some variables in a new Vite project. This happens because the default Eslint configuration requires prop-types in React projects. Here's how to resolve it:
```